Gemfile

```diff
@@ -70,6 +70,7 @@ gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
 gem 'uglifier', '>= 1.3.0'
 
 group :development, :test do
+  gem 'brakeman'
```

Comment:
We should move this into the lint group, or into a new dedicated group just for brakeman -- we can then use bundle-only to just install that group in the actions workflow. That should allow us to have a consistent version controlled by the gemfile.